firebase.json

```diff
@@ -181,6 +181,7 @@
       { "source": "/go/publishing-with-service-account", "destination": "/tools/pub/automated-publishing#publishing-from-google-cloud-build", "type": 301 },
       { "source": "/go/sdk-constraint", "destination": "/tools/pub/pubspec#sdk-constraints", "type": 301 },
       { "source": "/go/sdk-version-pinning", "destination": "https://github.com/dart-lang/sdk/wiki/Flutter-Pinned-Packages", "type": 301 },
+      { "source": "/go/security-advisories", "destination": "https://github.com/dart-lang/site-www/issues/5458", "type": 301 },
```

Comment:
Since the SDK could potentially have security advisories as well, could we make this more specific?

```suggestion
      { "source": "/go/pub-security-advisories", "destination": "https://github.com/dart-lang/site-www/issues/5458", "type": 301 },
```

Reply:
I didn't yet land the place where I am using it. I like your suggestion, so I'll make a new PR.